Flow.Launcher/Languages/fr.xaml

```diff
@@ -16,62 +16,62 @@
-    <system:String x:Key="PositionReset">Position Reset</system:String>
-    <system:String x:Key="PositionResetToolTip">Reset search window position</system:String>
+    <system:String x:Key="PositionReset">Réinitialiser la position</system:String>
+    <system:String x:Key="PositionResetToolTip">Rétablir la position de la fenêtre de recherche</system:String>
 
     <!--  Setting General  -->
     <system:String x:Key="flowlauncher_settings">Paramètres</system:String>
     <system:String x:Key="general">Général</system:String>
     <system:String x:Key="portableMode">Mode Portable</system:String>
     <system:String x:Key="portableModeToolTIp">Stocker tous les paramètres et données utilisateur dans un seul dossier (Pratique en cas d'utilisation de disques amovibles ou de services cloud).</system:String>
     <system:String x:Key="startFlowLauncherOnSystemStartup">Lancer Flow Launcher au démarrage du système</system:String>
-    <system:String x:Key="setAutoStartFailed">Error setting launch on startup</system:String>
+    <system:String x:Key="setAutoStartFailed">Erreur lors de la configuration du lancement au démarrage</system:String>
     <system:String x:Key="hideFlowLauncherWhenLoseFocus">Cacher Flow Launcher lors de la perte de focus</system:String>
     <system:String x:Key="dontPromptUpdateMsg">Ne pas afficher les notifications lors d'une nouvelle version</system:String>
-    <system:String x:Key="SearchWindowPosition">Search Window Position</system:String>
-    <system:String x:Key="SearchWindowScreenRememberLastLaunchLocation">Remember Last Position</system:String>
-    <system:String x:Key="SearchWindowScreenCursor">Monitor with Mouse Cursor</system:String>
-    <system:String x:Key="SearchWindowScreenFocus">Monitor with Focused Window</system:String>
-    <system:String x:Key="SearchWindowScreenPrimary">Primary Monitor</system:String>
-    <system:String x:Key="SearchWindowScreenCustom">Custom Monitor</system:String>
-    <system:String x:Key="SearchWindowAlign">Search Window Position on Monitor</system:String>
-    <system:String x:Key="SearchWindowAlignCenter">Center</system:String>
-    <system:String x:Key="SearchWindowAlignCenterTop">Center Top</system:String>
-    <system:String x:Key="SearchWindowAlignLeftTop">Left Top</system:String>
-    <system:String x:Key="SearchWindowAlignRightTop">Right Top</system:String>
-    <system:String x:Key="SearchWindowAlignCustom">Custom Position</system:String>
+    <system:String x:Key="SearchWindowPosition">Position de la fenêtre de recherche</system:String>
+    <system:String x:Key="SearchWindowScreenRememberLastLaunchLocation">Se souvenir de la dernière position</system:String>
+    <system:String x:Key="SearchWindowScreenCursor">Surveiller avec le curseur de la souris</system:String>
+    <system:String x:Key="SearchWindowScreenFocus">Surveiller avec la fenêtre ciblée</system:String>
+    <system:String x:Key="SearchWindowScreenPrimary">Écran principal</system:String>

```

Comment:
Google translater gives the word "moniteur" for the word "monitor"

Reply:
Sorry what do you mean? I translated with Google, seem ok.

Reply:
Sorry. This looks good. I said that here "Écran" gives the meaning of "Screen". Same thing btw